spa/src/components/BanCheck.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+<div>
+  <div style="font-size: small" v-if="checking">Checking for Bans...</div>
+</div>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+
+export default Vue.extend({
+  name: "BanCheck",
+  data: () => {
+    return {
+      checking: true,
+    };
+  },
+  methods: {
+    async isBanned(): Promise<any> {
+      return this.$http.post("/member/is_banned")
+        .then((response) => {
+          if (response.data.data === 0) {
+            this.$store.methods.destroySession();
+            this.$router.push({name: "login"});
```

Comment:
I'd suggest a new route to redirect the banned user to which states their account has been suspended. We can then use this page in the future for directions on disputes.